lib/select.rb

```diff
@@ -0,0 +1,59 @@
+require_relative 'menu.rb'
+
+class Select
+
+	attr_reader :order, :menu, :due
+
+	def initialize
+		@menu 	= Menu.new
+		@menu 	= @menu.menu
+		@order 	= {}
+		@total 	= []
+		@due 	= 0
+	end
+
+	def see
+		@menu
+	end
+    
+    def pick(dish , quantity=1)
+    	string_support(dish)
+    	fail 'this dish is not in the menu' unless @menu.include? dish
+    	set_order_total(dish , quantity)
+    end
+
+    def bill
+    	calculadora
+    	"#{@order} +'\n' + Due #{@due}£"
+    end
+
+    def place_order
+    	bill
+    	deliver
+    end
+
+
+private
+
+	def set_order_total(dish , quantity)
+		@order.merge!({"#{quantity} #{dish}"=>{"#{@menu[dish]}£ each"=>"#{quantity*@menu[dish]}£"}})
```

Comment:
Metrics/LineLength: Line is too long. [94/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Performance/RedundantMerge: Use `@order["#{quantity} #{dish}"] = {"#{@menu[dish]}£ each"=>"#{quantity*@menu[dish]}£"}` instead of `@order.merge!({"#{quantity} #{dish}"=>{"#{@menu[dish]}£ each"=>"#{quantity*@menu[dish]}£"}})`. (https://github.com/JuanitoFatas/fast-ruby#hashmerge-vs-hash-code)<br>Style/BracesAroundHashParameters: Redundant curly braces around a hash parameter.